src/bfelookups.js

```diff
@@ -110,68 +110,68 @@ bfe.define('src/lookups/lcnames', ['require', 'exports', 'module', 'src/lookups/
         }
       }, 300); // 300 ms
     };
-  
+
     /*
-  
+
           subjecturi hasAuthority selected.uri
           subjecturi  bf:label selected.value
       */
     exports.getResource = lcshared.getResourceWithAAP;
   });
   bfe.define('src/lookups/lcshared', ['require', 'exports', 'module'], function (require, exports, module) {
     // require('https://twitter.github.io/typeahead.js/releases/latest/typeahead.bundle.js');
-  
+
     /*
           subjecturi propertyuri selected.uri
           selected.uri  bf:label selected.value
       */
-  
+
     exports.getResource = function (subjecturi, property, selected, process) {
       var triples = [];
-  
+
       var triple = {};
       triple.s = subjecturi;
       triple.p = property.propertyURI;
-      selected.uri = selected.uri;
+      // selected.uri = selected.uri;  // FIXME: I *think* this line is useless but feel too green to remove it

```

Comment:
is there any reason to keep this line?

Reply:
I think that self-referential line can be totally deleted. It does absolutely nothing. If you want to push another commit to remove it that would be ok with me.